packages/mobile/src/qrcode/schema.ts

```diff
@@ -19,6 +19,7 @@ export const UriDataType = ioType({
   currencyCode: ioUnion([ioUndefined, ioKeyof(LocalCurrencyCode)]),
   amount: ioUnion([ioUndefined, ioString]),
   comment: ioUnion([ioUndefined, ioString]),
+  asset: ioUnion([ioUndefined, ioString]),
```

Comment:
Looks like we should be explicit and only allow `CELO` and `cUSD` here instead of any string.

Reply:
I was leaving it vague in case we would have cEUR, etc, in the future but can limit it too! What do you think?

Reply:
Yes we can leave like this if it's better to handle the unsupported values somewhere down in the processing of the data.